src/shadowfax.py

```diff
@@ -1,6 +1,7 @@
 import asyncio
 from collections import defaultdict
 import copy
+import json
```

Comment:
This isn't needed, right?

Reply:
Relic of something from debugging. Removed now.